sdk/core/src/services/UserManagementService.ts

```diff
@@ -58,6 +59,7 @@ export default class UserManagementService {
   private _cognitoIdentityService
   private _keyStorageApiService
   private _sessionStorageService
+  private _shouldDisableNameNormalisation = false
```

Comment:
but this default value only applies if you do not assign this field, and we're always assigning it?